presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorFactory.java

```diff
@@ -79,10 +79,6 @@ public Connector create(String catalogName, Map<String, String> config, Connecto
 
             return injector.getInstance(ThriftConnector.class);
         }
-        catch (InterruptedException ie) {
```

Comment:
Why these changes?

Reply:
@highker : Later version of `Boostrap` no longer  throw this exception. In fact we should backport https://github.com/prestosql/presto/pull/1549 to remove even Exception handling.